pkg/asset/machines/worker.go

```diff
@@ -163,8 +168,18 @@ func defaultKubevirtMachinePoolPlatform() kubevirttypes.MachinePool {
 	}
 }
 
+func defaultPowerVSMachinePoolPlatform() powervstypes.MachinePool {
+	return powervstypes.MachinePool{
```

Comment:
would be good to document that unlike AWS/Azure etc..there is no "instance type" that can be specified and so we have to define these explicitly.